CDCgov/prime-reportstream
# Remove replaceShorthand and update resolveConstant to return a list

This PR removes `replaceShorthand` and lets the FHIR engine do the work by ensuring the constants are passed to the appContext. It also updates `FhirPathCustomResolver.resolveConstant` to allow the return of multiple values, instead of just taking the first value.

Test Steps:
1. Run the following via postman and set the app to debug. See that `resolveConstant` returns the correct values for state (IG).
[Tests.postman_collection.txt](https://github.com/CDCgov/prime-reportstream/files/10701871/Tests.postman_collection.txt)

## Changes
- remove `replaceShorthand` function
- update `FhirPathCustomResolver.resolveConstant`

## Checklist

### Testing
- [x] Added tests? - I determined no additional unit tests were necessary, the existing tests cover everything related to `resolveConstant`

## Linked Issues
- Fixes #8106
